compiler/README.md

```diff
@@ -0,0 +1,70 @@
+<!-- README Document on how to build and use this Neural Network Compiler -->
+
+# Introduction
+
+This document discusses the ONE compiler. ONE compiler analyses, quantizes and optimizes the input model. It takes many model formats and converts to a common [Circle](#circle-format)  format which is then deployed.
+
+<!-- Contents -->
+# Contents 
+* [Build Setup](#Build-Setup)
+* [Supported Formats](#Supported-Nerual-Network-Formats)
+* [Model Analysis](#Model-Analysis)
+* [Data Representation & Quantization](#Data-Representation-And-Quantization)
+* [Circle Format](#Circle-Format)
+
+<!-- Build Setup -->
+## Build Setup
+<!-- System Requirments -->
+* System Setup   
+This setup is to build the ONE compiler on Ubuntu. Its tested on Ubuntu 16.04 as well as Ubuntu 18.04.  
+Get the code [Samsung ONE](https://github.com/Samsung/ONE) from Github. 
+
+<!-- Software/Package Requirements -->
+* Dependencies
+    * Cmake (Greater than or equal to 3.1)
+    * g++ toolchain ( Greater than or equal to 4.8)
+    * python virtual environment (python3-venv for python3)
+    * HD5 library (libhdf5-dev)
+
+<!-- Build Configuration -->
+* Configure  
+./nncc configure   
+
+You can pass CMAKE flags to above configuration command like (-DENABLE_TEST=1).  
+This would spit the missing packages if any errors.  
+
+
+<!-- Build -->
+* Build  
+./nncc build  
+After successful build, circle files are created inside build/compiler/*.circle.
+
+
+<!-- Test -->
+* Test  
+./nncc test  
+THis for now runs a set of tests.  
+TODO: Explanation on how to run a specific test.
```

Comment:
Right now, is it possible to run a single specific test?

Reply:
Compiler uses `gtest` for unit tests. Can you try `gtest` method to run a single unit test?

Reply:
Okay, will try with gtest and update this section later based on my try results. For first version, will skip this.